ipaserver/install/cainstance.py

```diff
@@ -1421,37 +1420,35 @@ def make_filter(dercert):
             ],
             ldap2.ldap2.MATCH_ALL)
 
-    def make_entry(dercert, entry):
-        cert = x509.load_der_x509_certificate(dercert)
+    def make_entry(cert, entry):
         serial_number = cert.serial_number
         subject = DN(cert.subject)
         issuer = DN(cert.issuer)
-        entry['usercertificate'].append(dercert)
+        entry['usercertificate'].append(cert)
```

Comment:
ever since 5a44ca638310913ab6b0c239374f4b0ddeeedeb3, `ipaldap` should be capable of dealing with the `cryptography.x509.Certificate` objects as well :)